webapp/package.json

```diff
@@ -63,7 +63,7 @@
   "resolutions": {
     "**/jquery": "3.2.x",
     "**/pouchdb-browser": "^7.2.2",
-    "**/moment": "^2.26.2"
+    "**/moment": "^2.29.1"
```

Comment:
Please check that [hindi patch works](https://github.com/medic/cht-core/blob/master/webapp/patches/moment-hindi-use-euro-numerals.patch) after this upgrade by selecting this language and see the display or error in console or in developer tools  :)